vector/v.out.ogr/testsuite/test_v_out_ogr.py

```diff
@@ -0,0 +1,166 @@
+"""Test of v.in.ogr
+
+@author Luís Moreira de Sousa
+"""
+
+from pathlib import Path
+from shutil import rmtree
+import grass.script as gs
+from grass.gunittest.case import TestCase
+
+
+class TestOgrExport(TestCase):
+
+    # Vector map in NC test dataset
+    test_map = "boundary_county"
+
+    # Result of import tests
+    temp_import = "test_ogr_import_map"
+
+    # Temporary location
+    temp_location = "temp_location"
+
+    # Temporary Homolosine map
+    temp_54052 = "temp_rand.gpkg"
+
+    # Temporary environment file
+    session_file = ""
+
+    # Column on which to test v.univar
+    univar_col = "PERIMETER"
+
+    # Output of v.univar
+    univar_string = """n=926
+nmissing=0
+nnull=0
+min=9.64452
+max=3.70609e+06
+range=3.70608e+06
+sum=1.1223e+08
+mean=121199
+mean_abs=121199
+population_stddev=342855
+population_variance=1.17549e+11
+population_coeff_variation=2.82886
+sample_stddev=343040
+sample_variance=1.17676e+11
+kurtosis=33.681
+skewness=4.86561
+"""
+
+    @classmethod
+    def setUpClass(cls):
+        """Use temporary region settings"""
+        cls.use_temp_region()
+
+    @classmethod
+    def tearDownClass(cls):
+        """!Remove the temporary region"""
+        cls.del_temp_region()
+
+    def tearDown(self):
+
+        # Remove maps in temp mapset
+        self.runModule(
+            "g.remove", type="vector", flags="f", pattern=f"{self.temp_import}*"
+        )
+        # Remove temporary files
+        for p in Path(".").glob(f"{self.test_map}*"):
+            p.unlink()
+        for p in Path(".").glob(f"{self.temp_54052}*"):
+            p.unlink()
+        if len(self.session_file) > 0:
+            Path(self.session_file).unlink()
+
+        # Remove temporary location
+        env = gs.parse_command("g.gisenv")
+        dbase = env["GISDBASE"].replace("'", "").replace(";", "")
+        rmtree(f"{dbase}/{self.temp_location}", ignore_errors=True)
+
+    def test_1(self):
```

Comment:
Please name the test according to the tested format or whatever this is testing. Short one-sentence docstring would be nice, but not needed.

Reply:
I added a new commit with these changes.